README.md

```diff
@@ -44,3 +44,23 @@ It is possible to use a `substra-backend` skaffold file as the source for the ne
 ```
 SUBSTRA_TESTS_SKAFFOLD_FILEPATH=$SUBSTRA_SOURCE/substra-backend/skaffold.yaml make test
 ```
+
+# Test design guidelines
+
+When adding or modifying tests, please follow these guidelines.
+
+The test suite:
+- Must be independent from the substra network
+  - The substra network is started prior to, and independently from the tests being run
```

Comment:
What about "The substra network **must** be started prior to (and independently from) **running the tests**" (without the bold)?

Rationale:

- "must" in "network must be started" might make it clearer that it's a prerequisite/requirement
- "running the tests" reads easier than "the tests being run"

Reply:
Looks better.
I changed running to executing to avoid redundancy with the next sentence.